selftests/functional/test_fetch_asset.py

```diff
@@ -57,7 +57,7 @@ def test_asset_fetch_find_success(self):
         print(foo)
         """ % (assetname, url)
         test_content = TEST_TEMPLATE.format(content=fetch_content)
-        test_file = tempfile.NamedTemporaryFile(suffix=".py", delete=False)
+        test_file = tempfile.NamedTemporaryFile(suffix=".py", prefix="avocado-fetchasset", delete=False)
```

Comment:
Like in the previous commit, it may make sense to simply change those classes to inherit from `TestCaseTmpDir`.